plugins/transport-grpc/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@amplication/plugin-transport-grpc",
+  "version": "0.0.1",
+  "description": "set grpc as protocol for Amplication build",
+  "main": "dist/index.js",
+  "scripts": {
+    "dev": "webpack --watch",
+    "build": "webpack",
+    "prebuild": "rimraf dist",
+    "format:write": "prettier --write \"**/*.{js,ts}\""
+  },
+  "author": "Mor Hagbi",
+  "license": "ISC",
+  "devDependencies": {
+    "@amplication/code-gen-types": "2.0.13-beta.1",
```

Comment:
I think the code-gen-types should move from devDependencies into dependencies. @overbit please opine on this.